src/utils/rum.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+const LIBRARY_LOCATION = 'https://www.hlx.live/tools/sidekick/library/index.js';
```

Comment:
I would not do that, but use the current URL instead.

Reply:
@trieloff Just waiting to hear back from David on what his expectation was here.

What I was thinking is that this RUM collection was more for us to understand usage of the sidekick library rather than tracking for the customer. So that's why I hardcoded the referer to the path of application on hlx.live.

If that is the case, I'm assuming using window.location.href as the referer is not what we want here. Wdyt?

Reply:
The `referer` property is used as the current URL in practice. We will be able to distinguish the percentage of library vs. sidekick usage by looking at the checkpoints alone.

Reply:
k, thanks for confirming @trieloff